docker/atlassian.yml

```diff
@@ -57,6 +57,12 @@ services:
             - "8085"
         networks:
             - artemis
+        healthcheck:
```

Comment:
The container didn't start on my machine in 240s. It turns out is asks for license:
<img width="1504" alt="image" src="https://github.com/ls1intum/Artemis/assets/32958017/6a236e58-9a0e-45e9-acb4-129b9b1efbe7">

Do I miss any config steps?

Reply:
Oh, I didn't think of the initial setup, then the healthcheck does not work. I will increase the retries to 120 so that after around 20 minutes (I think that should be enough time to set bamboo up following the docs), bamboo is set to unhealthy if the API is still not available.

Reply:
This looks fine, thanks